internal/ad/adsys-gpolist

```diff
@@ -234,6 +234,9 @@ def main():
     # Users don’t need @, as we already have the specific-domain ticket
     if args.objectclass == ObjectClass.user:
         accountname = accountname.split('@')[0]
+    else:
+        # Removes the domain part on FQDN (hostname.domain) hostnames from the account name
+        accountname = accountname.split('.')[0]
```

Comment:
Are you sure this is the right place for this fix? Shouldn’t it be at the same level when we normalize the user name in the Go code itself? I feel that we may store some cache with FQDN when we should not…

Reply:
It could be (but then why are we only cutting off the domain here?). I basically tracked the error message that the user reported and, when I've seen the if with the split at @, I thought that this script could be run directly.

Reply:
> (but then why are we only cutting off the domain here?)
When we normalize the user, we normalize the user to `user@domain.com` and this is what is used to store all caches and so on (as an user can be in multiple domain). So, the is the normalized name.

This scripts though (through samba) as the domain as a parameter and won’t accept the user in this form, this is why we are stripping it.

The machine is only part of one domain, this is why the cached files on disk should not contain the domain.

> I thought that this script could be run directly.
No, it’s embeeded in Go code and executed only by it (it’s even not accessible for the user apart from explicitely requesting it for debugging purposes)

Reply:
I'm trying to set up a test for it, as it appears we don't have any